docs/reference/commandline/service_create.md

```diff
@@ -875,6 +876,14 @@ $ docker inspect --format="{{.Config.Hostname}}" 2e7a8a9c4da2-wo41w8hg8qanxwjwsg
 x3ti0erg11rjpg64m75kej2mz-hosttempl
 ```
 
+### Specify isolation (Windows)
+
+By default, tasks scheduled on Windows nodes are run using the default isolation mode configured for this particular node. If you want to force a specific isolation, you can use the `--isolation` flag (valid values are `default`, `process` and `hyperv`): 
+```bash
```

Comment:
Nit: you'll need a newline before the  code block.

I'd rewrite this so that the values are listed out with their meanings, rather than just being within parentheses inline.